### PR TITLE
Navigation Bar

### PR DESCRIPTION
**Pull request by shayla11**
This pull request holds the code for implementing the navigation bar on all pages of the website. I did this by adding the styles to each pages CSS file. (To do this I had to create a style sheet for projects.html). This is the first of my plan to enhance my portfolio.

### CODE REVIEW

File: portfolio/src/main/webapp/about.html [hunk 1]
```diff
@@ -7,7 +7,13 @@
...
    </head>
 
-   <body class="bgcolor">
+   <body>
+      <ul>
+		<li><a href="index.html">Home</a></li>
+		<li><a href="about.html">About Me</a></li>
+		<li><a href="projects.html">Projects</a></li>
+		<li><a href="#Blog">Blog</a></li>
+	  </ul>
```
Comment (oydang) [inline portfolio/src/main/webapp/about.html:16]: Nit: This section doesn't seem to be indented correctly.

File: portfolio/src/main/webapp/projects.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
+.title {
+	text-align: center;
+	margin-bottom: 30px;
+}
+
+p {
+    text-align: center;
```
Comment (oydang) [inline portfolio/src/main/webapp/projects.css:7]: Nit: indentation looks wrong

File: portfolio/src/main/webapp/projects.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.gifsize {
+	width: 200px;
+	height: 200px;
+	margin-left: 200px;
+}
+
+.image {
+    grid-area: image;
```
Comment (oydang) [inline portfolio/src/main/webapp/projects.css:17]: Nit: indentation looks wrong

File: portfolio/src/main/webapp/projects.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+.grid-container {
+	display: grid;
+	grid-template-areas:
+		'image project project project project project '
+		'image footer footer footer footer footer';
+	padding: 25px;
+}
+
+.grid-container>div {
```
Comment (oydang) [inline portfolio/src/main/webapp/projects.css:39]: Nit: space between the elements: `.grid-container > div`. I think you should find a good auto-format solution that can automatically do this... how is VS Code's CSS formatter?

File: portfolio/src/main/webapp/projects.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+li a {
+	display: block;
+	color: black;
+	text-align: center;
+	padding: 14px 16px;
+	text-decoration: none;
+}
+
+li a:hover {
+	background-color: #A0AAFC;
```
Comment (oydang) [inline portfolio/src/main/webapp/projects.css:64]: Nit: lowercase background color

File: portfolio/src/main/webapp/projects.css [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+.grid-container>div {
+	font-size: 20px;
+}
+
+ul {
+	list-style-type: none;
+	margin: 0;
+	padding: 0;
+	overflow: hidden;
+	background-color: #E8EAFF;
```
Comment (oydang) [inline portfolio/src/main/webapp/projects.css:48]: Nit: lowercase background color

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -18,3 +18,45 @@
     display:table;
     margin: 0 auto;
 }
+
+ul {
+	list-style-type: none;
+	margin: 0;
+	padding: 0;
+	overflow: hidden;
+	background-color: #E8EAFF;
```
Comment (oydang) [inline portfolio/src/main/webapp/style.css:27]: Nit: lowercase background color

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -18,3 +18,45 @@
...
+li a {
+	display: block;
+	color: black;
+	text-align: center;
+	padding: 14px 16px;
+	text-decoration: none;
+}
+
+li a:hover {
+	background-color: #A0AAFC;
```
Comment (oydang) [inline portfolio/src/main/webapp/style.css:52]: Nit: lowercase color

File: portfolio/src/main/webapp/about.css [hunk 9]
```diff
@@ -123,3 +123,31 @@ p {
...
+	text-decoration: none;
+}
+
+li a:hover {
+	background-color: #A0AAFC;
+}
+
+body {
+	background-color: #f8f9ff;
+}
```
Comment (oydang) [inline portfolio/src/main/webapp/about.css:153]: Nit: newline at end of every file

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (shayla11) [reply portfolio/src/main/webapp/about.html:16]: (I accidentally made a "review" when I meant to just make a comment, sorry if there was any confusion!)

Screenshot 2020-06-08 at 1 21 56 PM - Display 2 - Edited

On my end, I'm not seeing the same spacing errors. I'm not sure were the problem is stemming from, whether its me, cloud, or github.

File: portfolio/src/main/webapp/projects.css (hunk 4 above)
Comment (shayla11) [reply portfolio/src/main/webapp/projects.css:39]: Absolutely! The VS gives me the same error that there is "no formatter". Ill try again to see if I can find a different solution to enabling it!

Comment (shayla11) [reply portfolio/src/main/webapp/projects.css:39]: Good news! I'm not getting an error when I run the auto format in VS anymore! I'm going to do another commit with the new formatting. This seems correct, when I cross reference with the actual guide! 

I'm not sure what solved the problem, maybe it was me closing it and restarting again, but hopefully this is the start of ALOT less formatting errors!

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (oydang) [reply portfolio/src/main/webapp/about.html:16]: Oh interesting. Is that screenshot from Cloud Shell? My guess is the GitHub branch isn't updated with the correctly formatted changes. Though Cloud Shell could also **not** be saving the file correctly. :(

File: portfolio/src/main/webapp/index.html [hunk 10]
```diff
@@ -53,7 +10,7 @@
 
 <body>
```
Comment (oydang) [inline portfolio/src/main/webapp/index.html:11]: Nit: Can you try auto-formatting this file? The indentations look much bigger to me than your other HTML files.

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (shayla11) [reply portfolio/src/main/webapp/about.html:16]: Yes its from cloud shell. I'll manually save the file just to see what happens.

**Pull request merged**